component-lib/snackbar.css

```diff
@@ -0,0 +1,66 @@
+.snackbar {
+	background-color: var(--bg-color-primary);
+	color: var(--primary-font-color);
+	max-width: 40rem;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	font-size: 1.4rem;
+	padding: 1rem;
+	gap: 1rem;
+	border-radius: 5px;
+	z-index: 1;
+}
+.snackbar-action {
+	width: 4rem;
+	height: 4rem;
+	border-radius: 50%;
+	border: 0;
+	background-color: transparent;
+	color: inherit;
+}
+.snackbar-action:hover {
+	cursor: pointer;
+}
+.snackbar-primary {
+	background-color: #adeef1;
+	color: #005d89;
+	border-color: #005d89;
+}
+.snackbar-secondary {
+	background-color: #ebeaed;
+	color: #686a8c;
+	border-color: #686a8c;
```

Comment:
If you are planning to use these colors through out component library, keeping var color in root will be more manageable, otherwise this is fine.

Reply:
I am using here only, so I did not add variables. I will try to change these colors to avariables.